inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/activation.cpp

```diff
@@ -69,8 +69,8 @@ const std::map<ActivationTypes, std::vector<std::vector<float>>> intActivationTy
 };
 
 const std::map<ActivationTypes, std::vector<std::vector<float>>> activationParamTypes = {
-    {PReLu, {{-0.01f}}},
-    {LeakyRelu, {{0.01f}}}
+        {PReLu, {{}}},
```

Comment:
It might be a good idea to leave a comment on how the slope values will be generated.

Reply:
Sure, done.